rental.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'book'
+require_relative 'person'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- I suggest that you make a `book` and `person` `attr_accessor` as well because we want to access and modify them

Reply:
resolved